blog/2025-03-17.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Call for Posters: Use Cases for the New Data Management Service Platform"
+authors: fuqua
+tags: [architecture, tanager]
+---
+
+[Tech Congress 2025](https://events.bizzabo.com/techcongress25) is just over a
+month away and [Project Tanager Milestone 0.4.0](./2025-03-13.md) has been
+reached. Can you help us show off what it is (or might be) capable of?
+
+Think of this as a call for posters. If you're not familiar with academic
+conferences and posters... then maybe you can imagine a science fair board. Do
+something cool, show it off in a few words and some pictures, and you get a few
+minutes to narrate for your audience. Complete your "poster" before
+April 17 so that I have time to incorporate it into a slide deck for an April 23
+Tech Congress session showing off the Data Management Service in Action.
+
+<!-- truncate -->
+
+:::tip
+
+Accepting prototype concepts without POC implementations! So long as the idea is
+architecturally sound.
+
+:::
+
+## Format
+
+Please share up to ~5 slides showcasing your idea. They might look like this:
+
+* Use case / user story.
+* Architecture diagram(s), if any additional components are needed to complement
+  the reference architecture in Docker Compose.
+* Ideally - but not required - screenshots showing a system in action, no matter
+  how rough looking it may be.
+
+Note that a live demonstration will not be possible from the front of the room
+during the Tech Congress session. Very short videos may be accepted, though we
+don't know in advance how well any audio will work.
+
+## Context
+
+Before getting to the ideas, let's review what is in milestone 0.4.0:
+
+* Two ASP.NET applications:
+  * Data Management Service (DMS):
+    * [Ed-Fi Discovery API](/reference/data-exchange/api-guidelines/design-and-implementation-guidelines/api-design-guidelines/discovery-api)
+      * Including Open API specifications.
+    * [Ed-Fi Resources API](/reference/data-exchange/api-guidelines/design-and-implementation-guidelines/api-design-guidelines/resources/)
+      * Data Standard 5.2 with TPDM 1.0.
+    * [Ed-Fi Descriptors API](/reference/data-exchange/api-guidelines/design-and-implementation-guidelines/api-design-guidelines/ed-fi-descriptors)
+  * DMS Configuration Service:
+    * Mostly compatible with the [Admin API 2.2 specification](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-API-Standards/blob/main/api-specifications/admin-api/admin-api-2.2.0.yaml).
+* Storage:
+  * PostgreSQL for modifications
+  * OpenSearch or Elasticsearch for queries.
+* Event streaming in Kafka.
+* Authorization (see note):
```

Comment:
We haven't used these terms before. I'm not entirely clear on the difference between resource authorization and document authorization. In DMS backend, everything is treated as documents, while the DMS API treats everything as resources.

Reply:
For the DMS, these terms are [defined here](https://github.com/Ed-Fi-Alliance-OSS/Project-Tanager/blob/main/docs/DMS/AUTHORIZATION/README.md)

`/ed-fi/students` is a resource that retrieves many documents.